tests/general/test_jnp_vs_np.py

```diff
@@ -0,0 +1,235 @@
+import pytest
+import numpy as np
+import scipy.fftpack
+
+import jax.numpy as jnp
+from jax import random, device_put
+import jax.config as config 
+
+key = random.PRNGKey(0)
+
+
+def subsample_fourier(x, k):
+  N = x.shape[-1]
+  res = x.reshape(x.shape[:-1] + (k, N // k)).mean(axis=(-2,))
+  return res
+
+def np_pad(x, pad_left, pad_right):
+  res = np.pad(x, ((0, 0), (0, 0), (pad_left, pad_right),), mode='reflect')
+  return res
+
+def jnp_pad(x, pad_left, pad_right):
+  res = jnp.pad(x, ((0, 0), (0, 0), (pad_left, pad_right),), mode='reflect')
+  return res
+
+def test_multiplication():
+    # multiplication
```

Comment:
Remove these intro comments in each method.

Reply:
ah lol yeah